portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,49 +31,81 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+
+@WebServlet("/new-post")
 public class DataServlet extends HttpServlet {
  private ArrayList<String> check; 
  public void init() {
+
     check = new ArrayList<String>();
      check.add("Mood");
   check.add("Summer");
     check.add("Style");
   check.add("WFH");
     check.add("Experience");
   check.add("Awesome!");
+
 }
 
+ 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-  //Initializing a gson instance
-  Gson gson = new Gson();
-  //Converting the Arraystring to a Json. 
-    String json = gson.toJson(check);
-   
-    response.setContentType("text/html;");
-    response.getWriter().println(json);
+  response.setContentType("text/html;");
+    response.setContentType("application/json;");
     
+    Query query = new Query("Messages").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    PreparedQuery results = datastore.prepare(query); 
+
+ 
+
+    
+ 
```

Comment:
fix spacing and remove extra newlines (at most one between blocks) for more consistency.

Reply:
I agree.